sonic_db_config/db_config_test.go

```diff
@@ -0,0 +1,78 @@
+package dbconfig
+
+import (
+	"github.com/Azure/sonic-telemetry/test_utils"
+	"testing"
+)
+
+func TestGetDb(t *testing.T) {
+	t.Run("Id", func(t *testing.T) {
+		db_id := GetDbId("CONFIG_DB", GetDbDefaultNamespace())
+		if db_id != 4 {
+			t.Fatalf(`Id("") = %d, want 4, error`, db_id)
+		}
+	})
+	t.Run("Sock", func(t *testing.T) {
+		sock_path := GetDbSock("CONFIG_DB", GetDbDefaultNamespace())
+		if sock_path != "/var/run/redis/redis.sock" {
+			t.Fatalf(`Sock("") = %q, want "", error`, sock_path)
+		}
+	})
+	t.Run("AllNamespaces", func(t *testing.T) {
+		ns_list := GetDbAllNamespaces()
+		if len(ns_list) != 1 {
+			t.Fatalf(`AllNamespaces("") = %q, want "1", error`, len(ns_list))
+		}
+		if !(ns_list[0] == GetDbDefaultNamespace()) {
```

Comment:
`!(ns_list[0] == GetDbDefaultNamespace())` this can be simplified with operator `!=`

Reply:
updated